packages/manager-react-components/src/components/wrap/__tests__/Wrap.spec.tsx

```diff
@@ -0,0 +1,95 @@
+import { render, screen } from '@testing-library/react';
+import { vi } from 'vitest';
+import { Wrap } from '../Wrap.component';
+import { WrapPreset } from '../Wrap.props';
+
+vi.mock('@ovhcloud/ods-react', () => ({
+  Text: ({ children, preset, className, ...props }: any) => (
+    <div
+      data-testid="ods-text"
+      data-preset={preset}
+      className={className}
+      {...props}
+    >
+      {children}
+    </div>
+  ),
+}));
+
+describe('Wrap Component', () => {
+  const defaultText = 'Test content';
+
+  describe('Rendering', () => {
+    it('should render children content', () => {
+      render(<Wrap>{defaultText}</Wrap>);
+      expect(screen.getByText(defaultText)).toBeInTheDocument();
+    });
+
+    it('should render with custom children', () => {
+      const customContent = 'Custom test content';
+      render(<Wrap>{customContent}</Wrap>);
+      expect(screen.getByText(customContent)).toBeInTheDocument();
+    });
+
+    it('should render complex children', () => {
+      const complexContent = (
+        <div>
+          <span>Nested content</span>
+          <strong>Bold text</strong>
+        </div>
+      );
+      render(<Wrap>{complexContent}</Wrap>);
+      expect(screen.getByText('Nested content')).toBeInTheDocument();
+      expect(screen.getByText('Bold text')).toBeInTheDocument();
+    });
```

Comment:
This is mostly testing the React's children behavior and not specific to Wrap component at all.